deploy/tests/000032-upgrade-bancor-portal-v2.ts

```diff
@@ -0,0 +1,27 @@
+import { BancorPortal, ProxyAdmin } from '../../components/Contracts';
+import { expectRoleMembers, Roles } from '../../test/helpers/AccessControl';
+import { describeDeployment } from '../../test/helpers/Deploy';
+import { DeployedContracts } from '../../utils/Deploy';
+import { expect } from 'chai';
+import { getNamedAccounts } from 'hardhat';
+
+describeDeployment(__filename, () => {
+    let deployer: string;
+    let proxyAdmin: ProxyAdmin;
+    let bancorPortal: BancorPortal;
+
+    before(async () => {
+        ({ deployer } = await getNamedAccounts());
+    });
+
+    beforeEach(async () => {
+        proxyAdmin = await DeployedContracts.ProxyAdmin.deployed();
+        bancorPortal = await DeployedContracts.BancorPortal.deployed();
+    });
+
+    it('should deploy and configure the bancor portal contract', async () => {
+        expect(await proxyAdmin.getProxyAdmin(bancorPortal.address)).to.equal(proxyAdmin.address);
+        expect(await bancorPortal.version()).to.equal(2);
+        await expectRoleMembers(bancorPortal, Roles.Upgradeable.ROLE_ADMIN, [deployer]);
+    });
+});
```

Comment:
We should add fork-only basic portal usage here and in `network.ts`. The test here is only supposed to do a quick sanity check that the upgraded portal works (e.g., by using it to migrate some positions), while the test in `network.ts` is the one that is going to be executed every 10 minutes and maybe more extensive.

IMHO, if you'll add a "good" test to `network.ts` (use the whales to create a both ETH and TKN positions for each protocol and then use the portal to migrate them to Bancor), then adding a similar test to `000032-upgrade-bancor-portal-v2.ts` **is not required**.

Reply:
Resolving and will focus on `network.ts`